### DISCUSSION TITLE
Invalid token and other questions

### DISCUSSION THREAD

**Discussion by tvolse2** [netbirdio/netbird#1171] [Q&A / Support]
Hello,
I have netbird installed in docker together with authentication via Zitadel. Now it happens to me that the login via netbird stops working and it shows the error 401 request failed token invalid. It will only help to restart the docker container in which netbird is running. Don't know what it can do? It happens about every two days now.
The following error appeared in the log:
**/server/http/middleware/auth_middleware.go:65: Error when validating JWT claims: Error parsing token: unable to find appropriate key**

I also discovered that a user with the email address: 232122293172568068 appeared in the netbird web administration and it is not even possible to block it. Is it possible that he could somehow register there? I will also see the following information in the log:

**ERRO management/server/telemetry/http_api_metrics.go:181: HTTP response 2638241688: PUT /api/users/232122293172568068 status 404**

I also see the following information in the log:
**grpcserver.go:326: failed logging in peer +TIzh/yKTQ3euySbqoP9h6SrjZJO/iMflgWOl3PG1Sw=**

Are these attempts to log in under the wrong user?

I still wanted to ask what this information in the log means:

**/server/account.go:1335: overriding JWT Domain and DomainCategory claims since single account mode is enabled**

I know this is a lot of questions, but thank you very much for the answers.

**Comment by braginini** [comment]
Hey @tvolse2 
Sorry to hear that you have these issues. 
Would you mind joining slack and pinging us there in the general channel referring to this issue? We are much faster there 
https://join.slack.com/t/netbirdio/shared_invite/zt-vrahf41g-ik1v7fV8du6t0RwxSrJ96A

The last log line about the single account mode is just an informational message, indicating that all peers and users will be joining a single network.
